introduction-to-datasets.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
```

Comment:
I think you should also rephrase to get rid of the two "but" phrases in the second sentence.